Day1-adding_to_k/Brandon/adding.py

```diff
@@ -0,0 +1,28 @@
+# Given a list of numbers and a number k, return whether any two numbers from the list add up to k.
+# For example, given [10, 15, 3, 7] and k of 17, return true since 10 + 7 is 17.
+# Bonus: Can you do this in one pass?
+
+from copy import deepcopy
+
+def is_k_a_sum(list_of_nums,k):
+    # Using set because of the benefit that it provides in lookup with in operator
+    sum_set = set()
+    # Looping using the index and number to identify which number is itself
+    for index,number in enumerate(list_of_nums):
+        # copying to not modify the original
```

Comment:
enumerate is the "pythonic" way of doing `for i in range(len(list_of_nums)):`

enumerate is nicer because it will allow you to have the index and the current object.